bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -132,7 +132,7 @@ public void initiate() {
             metadataBuilder.withCustomMetadata(customMetadata);
         }
         if (bk.getConf().getStoreSystemtimeAsLedgerCreationTime()) {
-            metadataBuilder.withCreationTime(System.currentTimeMillis());
+            metadataBuilder.withCreationTime(System.currentTimeMillis()).storingCreationTime(true);
```

Comment:
This sounds like an unrelated bug.
Okay for keeping.
I wonder why we did not catch this bug on tests